app/src/main/java/ar/com/wolox/android/cookbook/scanqr/scanerror/ScanErrorFragment.kt

```diff
@@ -0,0 +1,28 @@
+package ar.com.wolox.android.cookbook.scanqr.scanerror
+
+import ar.com.wolox.android.cookbook.R
+import ar.com.wolox.android.cookbook.scanqr.ScanQrActivity
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import kotlinx.android.synthetic.main.fragment_scan_error.*
+import javax.inject.Inject
+
+class ScanErrorFragment @Inject constructor() : WolmoFragment<ScanErrorPresenter>(), ScanErrorView {
+
+    override fun layout(): Int = R.layout.fragment_scan_error
+
+    override fun init() {
+        presenter.attachView(this)
+        setListeners()
```

Comment:
It's not necessary to call `setListeners`. Wolmo does it instead

Reply:
I think that what is missing is `super.init()`

Reply:
The `super.init()` is an empty method, but the `setListener` should be invoked anyway.